baccmod/grid3d_acceptance_map_creator.py

```diff
@@ -322,7 +322,10 @@ def _create_base_computation_map(self, observations: Observations) -> Tuple[
         livetime = 0. * u.s
 
         with erfa_astrom.set(ErfaAstromInterpolator(1000 * u.s)):
-            for obs in observations:
+            for raw_obs in observations:
+                # Copy the observation to prevent modifyinf the original obs
```

Comment:
typo, and can be shortened to avoid abreviation:
# Copy the observation to prevent modifying the original